docs/BigQueryPushdownEngine-sqlengine.md

```diff
@@ -70,6 +73,18 @@ completed. This API can be used if the execution environment for this environmen
 Note that this API has an on-demand price model. See the [Pricing](https://cloud.google.com/bigquery/pricing#storage-api) 
 page for details related to pricing.
 
+**Attempt direct copy to BigQuery sinks**: Performance can be greatly improved if the records from stages that are executed using 
+BigQuery ELT Transformation Pushdown are copied directly into a configured table in a compatible BigQuery Sink. This eliminates
+the need to read records into the pipeline as no further processing is needed within the pipeline.
+In order to ensure this BigQuery Sink can take advantage of the performance improvements provided by this feature,
+the following need to be ensured:
```

Comment:
to avoid two ensure's in one sentence, how about something like "...the following requirements must be met:"